### DISCUSSION TITLE
iMX weston patch and release cycle?

### DISCUSSION THREAD

**Discussion by aalonso** [meta-flutter/meta-flutter#479] [Q&A] [answered]
Glad to see the scarthgap branch

Not familiar with meta-flutter release cycles, is there a place that this is documented ?

REAMDE.md also mentions:
- NXP iMX7, iMX8; imx-weston requires patch for ivi-homescreen + flutter-auto

where could that mentioned patch can be found?

**Comment by jwinarske** [comment, marked as answer]
Release cycles are generally around AGL release windows, or when working on a feature.  There is an upcoming AGL release at the end of next month, which will have ivi-homescreen/flutter-auto v2.

Here:
https://github.com/meta-flutter/meta-flutter/issues/295#issuecomment-1642841934

Or Here:
https://gerrit.automotivelinux.org/gerrit/gitweb?p=AGL/meta-agl-devel.git;a=blob;f=meta-agl-flutter/recipes-graphics/toyota/files/0001-Disable-on_frame_base_surface-wl_surface_commit.patch;h=a37233b50917f49ba561e6994c746d0a6373f4e5;hb=HEAD
